tools/samtools.wdl

```diff
@@ -462,9 +463,10 @@ task bam_to_fastq {
         String f = "0"
         String F = "0x900"
         String G = "0"
-        Boolean paired_end = true
+        Boolean append_read_number = true
         Boolean interleaved = false
         Boolean output_singletons = false
+        Boolean paired_end = true
```

Comment:
Just curious, did you reorder these to be alphabetical? The style-guide leaves it up to the author how to order parameters of the same type. My process for ordering things is typically:
1) follow the order of the underlying tools documentation
2) what feels most important for a user to see should be towards the top
    * obviously incredibly subjective here. But with this I'd say `paired_end` should be first in this block.

I'm fine with this being alphabetical, but I don't think any other inputs are. This will likely be the odd one out if left alphabetical, but we intentionally punted the question of "order once type order is satisfied", so I can't complain about anything you do here.

Reply:
I did choose alphabetical. That said, in this case, I think it makes sense as `paired_end` is an input for the task, where the other arguments more closely correspond with inputs to the underlying tool.

Reply:
Ok. The style-guide leaves it up to individual authors, so if you want to write tasks using alphabetic sort at this level, that's fine with me. But I'm going to continue using my (highly subjective) method of sorting inputs, so there will be some degree of inconsistency. I think that's fine. The fine-grain sorting here is fairly inconsequential.